kolena/workflow/annotation.py

```diff
@@ -77,6 +77,11 @@ class BoundingBox(Annotation):
     def _data_type() -> _AnnotationType:
         return _AnnotationType.BOUNDING_BOX
 
+    def area(self) -> float:
+        width = self.bottom_right[0] - self.top_left[0]
+        height = self.bottom_right[1] - self.top_left[1]
+        return width * height
```

Comment:
Would love a sanity check on if there's a specific reason we didn't want this attached to `kolena.workflow.BoundingBox`?